<ui version="4.0" >
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>36</y>
     <width>600</width>
     <height>342</height>
    </rect>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actHelpAbout" />
   </widget>
   <widget class="QMenu" name="mnuRecord" >
    <property name="title" >
     <string>&amp;Record</string>
    </property>
   </widget>
   <widget class="QMenu" name="mnuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="mnuView" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
   </widget>
   <addaction name="mnuFile" />
   <addaction name="mnuView" />
   <addaction name="mnuRecord" />
   <addaction name="mnuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>378</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>22</y>
     <width>600</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actFileSave" >
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDude" >
   <property name="text" >
    <string>Dude</string>
   </property>
  </action>
  <action name="actFileOpen" >
   <property name="text" >
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actFileSaveAs" >
   <property name="text" >
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actRecord" >
   <property name="text" >
    <string>&amp;Record</string>
   </property>
  </action>
  <action name="actFileNew" >
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actFileOpenRecent" >
   <property name="text" >
    <string>Open &amp;Recent</string>
   </property>
  </action>
  <action name="actFileLoadExampleProject" >
   <property name="text" >
    <string>Load &amp;Example Project</string>
   </property>
  </action>
  <action name="actFileExit" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actFileExportCSV" >
   <property name="text" >
    <string>Export as &amp;CSV File...</string>
   </property>
   <property name="iconText" >
    <string>Export as CSV File</string>
   </property>
  </action>
  <action name="actFileExportTAB" >
   <property name="text" >
    <string>Export as &amp;TAB File</string>
   </property>
  </action>
  <action name="action_About" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actHelpAbout" >
   <property name="text" >
    <string>&amp;About...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="View.qrc" />
 </resources>
 <connections/>
</ui>
